src/common/performance.js.flow

```diff
@@ -1,3 +0,0 @@
-declare export class performance {
```

Comment:
Because otherwise this happens during bundling:
`Failed to browserify: 'main' Cannot find module './performance' from '/Users/till/mozilla/servo/browserhtml/src/common'`

With the comments around flow types removed everywhere, this file is as much JS as everything else, so I don't see a reason not to do the renaming. In fact, I think we should just fold the contents of the `.flow` files into the respective `.js` files, but that's something for another PR.